### ISSUE TITLE
jinja2 url not using language when using with localeurl

### ISSUE THREAD

**Issue by GabLeRoux** [niwinz/django-jinja#52]
I started playing with django lately I tried using jinja2 instead of django's templating system, found django-jinja and it is very usefull and easy to install :)

I wanted to use [localeurl](https://bitbucket.org/carljm/django-localeurl) and I had troubles submitting my forms untill I removed `'localeurl.middleware.LocaleURLMiddleware',` from my `MIDDLEWARE_CLASSES`. In fact, `localeurl` introduces a redirect everytime you enter a url without language prefix (which is great), but because `django-jinja`'s `url` method doesn't care about localeurl's LocaleURLMiddleware, my forms are being redirected and they loose request parameters ie: `/form/` instead of `/en/form`, debug-tool bar then catch a redirect when the form is being sent.

**tldr;** would it be possible to write some kind of url method such as `locale_url('name')`? I liked to use jinja2 templates, but I prefer having the language redirect based on the browser's language with `localeurl`.

I am not sure if this issue should go here or on localeurl's repository, in my case, I moved back to django templates.
